print/ghostscript/Portfile

```diff
@@ -162,6 +180,17 @@ variant cups description {Enable CUPS driver} {
     configure.args-replace  --disable-cups --enable-cups
 }
 
+variant ghostpdl description {Install GhostPDL} {
+    configure.args-delete --without-pcl --without-xps --without-gpdl
+    notes "
+    The PCL interpreter requires access to the base 80 font set for proper
+    operation. To use the default URW fonts, add the following to your
+    shell profile:
+
+        export PCLFONTSOURCE='${prefix}/share/${name}/urwfonts/'
```

Comment:
How do you recommend I address the licensing issue?  Would a sentence in the notes be suitable?

> Note: These fonts are distributed under the AFPL licence which bars commercial use.

Reply:
I'm not really the best person to ask about license issues. You should check out the [guide about them](https://trac.macports.org/wiki/PortfileRecipes#license) however. I'm not sure if we need to do anything because Macports is not commercial.